framework/src/main/java/io/github/kgress/scaffold/webdriver/WebDriverManager.java

```diff
@@ -500,10 +498,17 @@ private RemoteWebDriver configureSauceRemoteBrowser(MutableCapabilities browserO
             if (sauce.getParentTunnel() != null) {
                 browserOptions.setCapability("parentTunnel", parentTunnel);
             }
-            browserOptions.setCapability("name", testName);
             if (remoteUrl == null) {
-                remoteUrl = URI.create("http://" + username + ":" + accessKey + sauceUrl).toString();
+                remoteUrl = URI.create("https://" + sauceUrl).toString();
             }
+
+            MutableCapabilities sauceCaps = new MutableCapabilities();
```

Comment:
[After reviewing the sauce labs doc](https://wiki.saucelabs.com/display/DOCS/W3C+Capabilities+Support) (under the "click here to see all capabilities specific to Sauce Labs link), it looks like we're going to have to add more than just these 4 capabilities to the new `sauce:options` object.

We don't necessarily need to add the entire list that sauce labs provides, but we should for sure add the other capabilities that are referenced on this method:
* parentTunnel
* tunnel

Is there anything else on that list that you think would be a good addition? It would be trivial to add them to `SauceContext`

Reply:
I've added parentTunnel and tunnel as suggested, good catch. I think the build and tags capabilities would be useful too but I'm not sure how to implement it.